django-backend/fecfiler/contacts/migrations/0011_alter_contact_id.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1.3 on 2023-01-03 21:51
+
+from django.db import migrations, models
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('contacts', '0010_auto_20220915_1309'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='contact',
+            name='id',
+            field=models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False),
```

Comment:
The CircleCI linter is reporting this line too long:
`./django-backend/fecfiler/contacts/migrations/0011_alter_contact_id.py:17:91: E501 line too long (106 > 90 characters)
`